pkg/gofr/datasource/pubsub/kafka/interfaces.go

```diff
@@ -22,4 +24,5 @@ type Connection interface {
 	Controller() (broker kafka.Broker, err error)
 	CreateTopics(topics ...kafka.TopicConfig) error
 	DeleteTopics(topics ...string) error
+	Close() error
```

Comment:
I would have expected a context here, even if not used

Same for reader

Reply:
it is not gofr interface but reflection of kafka's Connection type. Not doable